docs/getting_started.rst

```diff
@@ -62,6 +62,20 @@ the `Adafruit tutorial
 <https://learn.adafruit.com/character-lcds/wiring-a-character-lcd>`_ to learn
 how to wire up these circuits.
 
+Via pigpio
+~~~~~~~~~~
+
+If you decide to use the ``pigpio`` library to control the LCD, follow the
+instructions set out above. Please keep in mind that the ``pigpio`` can only
+use the BOARD numbering scheme.
```

Comment:
I think you meant to write BCM, right? https://raspberrypi.stackexchange.com/questions/12966/what-is-the-difference-between-board-and-bcm-for-gpio-pin-numbering

Reply:
Yes, you are right. Thanks for spotting that mistake. It should be sorted in the most recent change.